en/micro-integrator/docs/setup/deployment/k8s-pipeline/deploy-your-own-artifact.md

```diff
@@ -0,0 +1,86 @@
+# Deploy Your Own Artifact
+1.  Fork the [CICD sample](https://github.com/wso2-incubator/cicd-sample-docker-mi) repository which contains a [Maven
+    Multi-Module
+    project](https://ei.docs.wso2.com/en/latest/micro-integrator/develop/create-multi-maven-project/).
+
+2.  Update the repositories in the  [sample values](https://raw.githubusercontent.com/wso2/kubernetes-pipeline/master/samples/values-mi.yaml) file
+    used in the  [Quick Start Guide](/setup/deployment/k8s-pipeline/quick-start-guide/) to
```

Comment:
I think this link does not work. Please check all the links in this PR. I think it should just be (/quick-start-guide/).

Reply:
@samgnaniah : Relative URL refer from the root location since it has multilevel directories
it works fine.

Reply:
Yes it works for me